lib/index.js

```diff
@@ -915,10 +915,12 @@ MapboxGeocoder.prototype = {
       color: '#4668F2'
     }
     var markerOptions = extend({}, defaultMarkerOptions, this.options.marker)
-    this.mapMarker = new this._mapboxgl.Marker(markerOptions);
-    this.mapMarker
-      .setLngLat(selected.center)
-      .addTo(this._map);
+    if (selected.center) {
```

Comment:
This looks good. 

We could also try to compute the center from the `geometry.coordinates` if the center property doesn't exist. We should always have coordinates, as the results are expected to be geojson.

Reply:
Yes but since I don't think it can be guaranteed to always be a point geometry, eg. searching for a street might return the whole street linestring, or for a country might return the country polygon, so it can be a bit more complicated. That would be a future enhancement but for now I think this simple check should suffice.

Reply:
Okay I've added a extra commit which will check for a Point geometry and use that if exists.